gazebo/transport/TaskGroup.hh

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef _TASK_GROUP_HH_
+#define _TASK_GROUP_HH_
```

Comment:
some of the existing gazebo/transport/*.hh header guards follow this pattern, but I think it would be better as `GAZEBO_TRANSPORT_TASKGROUP_HH_`

Reply:
Done in https://github.com/osrf/gazebo/pull/3174/commits/f1fadd01d81ebd5cf4711b2314cea8c73eb74536 .